test/dataTypes.spec.ts

```diff
@@ -72,16 +75,34 @@ describe('RLP encoding (string):', function () {
 describe('RLP encoding (list):', function () {
   it('length of list 0-55 should return (0xc0+len(data)) plus data', function () {
     const encodedArrayOfStrings = RLP.encode(['dog', 'god', 'cat'])
+    assert.equal(RLP.getLength(encodedArrayOfStrings), 13)
     assert.equal(13, encodedArrayOfStrings.length)
     assert.equal(encodedArrayOfStrings[0], 204)
     assert.equal(encodedArrayOfStrings[1], 131)
     assert.equal(encodedArrayOfStrings[11], 97)
     assert.equal(encodedArrayOfStrings[12], 116)
   })
 
-  // it('length of list >55 should return 0xf7+len(len(data)) plus len(data) plus data', function () {
-  //   // need a test case here!
-  // })
+  it('length of list >55 should return 0xf7+len(len(data)) plus len(data) plus data', function () {
+    const data = [
+      'dog',
+      'god',
+      'cat',
+      'zoo255zoo255zzzzzzzzzzzzssssssssssssssssssssssssssssssssssssssssssssss',
+    ]
+    const encodedArrayOfStrings = RLP.encode(data)
+    assert.equal(RLP.getLength(encodedArrayOfStrings), 86)
```

Comment:
```suggestion
assert.strictEqual(RLP.getLength(encodedArrayOfStrings), 86)
```

Since `equal` is deprecated, can we update to the supported equivalent function?  Tests pass fine with this change.